powershell/generators/nuspec.ts

```diff
@@ -40,7 +40,7 @@ export async function generateNuspec(project: Project) {
     <file src="${removeCd(project.internalFolder)}/**/*.*" exclude="${removeCd(project.internalFolder)}/readme.md" />
     <file src="${removeCd(project.customFolder)}/**/*.*" exclude="${removeCd(project.customFolder)}/readme.md;${removeCd(project.customFolder)}/**/*.cs" />
     <file src="${removeCd(project.docsFolder)}/**/*.md" exclude="${removeCd(project.docsFolder)}/readme.md" />
-    <file src="${removeCd(project.exportsFolder)}/**/*.ps1" />
```

Comment:
No need to change this, right?

Reply:
Yes, we need. We do not need the separate ps1 file for each cmdlet, since we have combined them into one ps1 file to reduce signing work.

Reply:
Yes, but this will still capture the new file, and leave us with flexibility to make changes in the future without having to change this.